plenum/test/plugin/demo_plugin/test_plugin_basic.py

```diff
@@ -0,0 +1,49 @@
+from plenum.test.helper import waitForSufficientRepliesForRequests, \
+    checkReqNackWithReason
+from plenum.common.constants import CURRENT_PROTOCOL_VERSION, TXN_TYPE
+from plenum.common.request import Request
+from plenum.common.util import randomString
+from plenum.test.plugin.demo_plugin import AUCTION_LEDGER_ID, dummy_field_length
+from plenum.test.plugin.demo_plugin.constants import GET_BAL
+from stp_core.loop.eventually import eventually
+
+
+def test_plugin_setup(txnPoolNodeSet):
```

Comment:
Please make sure that we removed all old-style plugin code and tests

Reply:
Ok